aspnetcore/web-api/index.md

```diff
@@ -133,6 +133,10 @@ To customize the response that results from a validation error, use <xref:Micros
 
 [!code-csharp[](index/samples/2.x/Startup.cs?name=snippet_ConfigureBadRequestResponse&highlight=3-20)]
 
+### Log automatic 400 responses
+
+For information about how to log automatic HTTP 400 responses, see [issue #12157 in the AspNetCore.Docs repo](https://github.com/aspnet/AspNetCore.Docs/issues/12157).
```

Comment:
I propose we standardize our engineering issue/PR links. I suggest basing them on their titles (unless the titles are really bad). The following is the format that I've been using, but I'd like to adopt a format that the team agrees on and use it everywhere. I'll keep an :ear: out for guidance from the team.

```suggestion
For information on how to log automatic HTTP 400 responses, see [How automatically triggered model validation errors can be logged? (aspnet/AspNetCore #12157)](https://github.com/aspnet/AspNetCore.Docs/issues/12157).
```

Reply:
It's important to let people know they're linking out of the docs.  And if we include the title in the link text we should change the title to make it as effectively descriptive as possible within a reasonable length limit.

Reply:
Since we don't control them on the engineering side, there was a case or two where I had to use a different title. IIRC, most of the engineering issue titles were fine.

It's just a personal thing, but I've usually been less concerned on the *where-it-is* aspect. I just put that under the umbrella of *this is the Internet ... this is how the whole thing works*. :smile:

Reply:
Anywho ... let me know if you cats standardize. Otherwise, I'll keep going. I haven't had negative feedback from a reader about any of my cross-links ...... *yet* 🤞.